src/Microsoft.OpenApi.OData.Reader/Operation/ComplexPropertyBaseOperationHandler.cs

```diff
@@ -18,6 +18,7 @@ internal abstract class ComplexPropertyBaseOperationHandler : OperationHandler
     protected override void Initialize(ODataContext context, ODataPath path)
     {
         ComplexPropertySegment = path.LastSegment as ODataComplexPropertySegment ?? throw Error.ArgumentNull(nameof(path));
+        base.SetTargetPath();
```

Comment:
Perhaps `base.Initialize(context, path);` so that we can also `SetCustomLinkRelType()` while we are at it.